process.env

```diff
@@ -0,0 +1 @@
+SECRET = secret
```

Comment:
Remove process.env, use .env in root directory. Make sure all the require statements are back to using .env also.

Reply:
i'm not sure why but without specifying the process.env it the require statement just doesnt work for me? I guess I'll look into it more

Reply:
maybe changing the filename from process.env to .env would help??

Reply:
could you have a .env file without a prefix though .-. sounds kinda sketch

Reply:
lol .bash_profile .bash_history are all files that exists in a linux machine. They take care of some custom configurations, environment/path variables, and any commands that ran. A dotfile is usually for configuration sake and are meant to be hidden. Not sketch at all.